tmail-backend/apps/distributed-es6-backport/src/test/resources/rabbitmq.properties

```diff
@@ -1,2 +1,14 @@
 uri=amqp://james:james@rabbitmq_host:5672
-management.uri=http://james:james@rabbitmq_host:15672/api/
\ No newline at end of file
+management.uri=http://james:james@rabbitmq_host:15672/api/
+
+# AMQP resources parameters for the subscriber email address contact messages. In order to synchronize contacts between Tmail backend and 3rd.
+# AQMP uri
+address.contact.uri=amqp://rabbitmq:5672
+# Administration Management
+address.contact.management.uri=http://rabbitmq:15672
```

Comment:
What do we need the management URI for here?

Reply:
create the deadletter exchange, deadletter queue, and binding them

Reply:
I'm reconsidering this as well with my last comments... maybe some clients would not want TMail to have admin access to their RabbitMQ (but then it would mean they have the responsibility to create exchanges and queues and bindings).

We need to take a decision on where we stand here I think?

Reply:
So yeah after talking with @chibenwa we don't need management uri and creds indeed. It seems the normal uri is enough to create the queues and exchanges. In terms of security it's not a good practice either.

Can you remove those please?

Reply:
We need the creds... just not the management uri.. I'm rusty on RabbitMQ ><